pgjdbc/src/test/java/org/postgresql/core/ParserTest.java

```diff
@@ -114,6 +114,24 @@ public void testSelectCommandParsing() {
     assertTrue("Failed to correctly parse lower case command.", Parser.parseSelectKeyword(command, 0));
   }
 
+  /**
+   * Test SET command parsing.
+   */
+  @Test
+  public void testSetCommandParsing() {
+    char[] command = new char[3];
+    "SET".getChars(0, 3, command, 0);
+    assertTrue("Failed to correctly parse upper case command.", Parser.parseSetKeyword(command, 0));
```

Comment:
Please use parameterized test or extract this into a method so the logic is not duplicated, and the assertion message includes the input data.

See https://github.com/pgjdbc/pgjdbc/blob/master/CONTRIBUTING.md#test

Reply:
For sure, done